README.md

```diff
@@ -46,8 +46,8 @@ $ make bootstrap
 
 If everything went well, you should now be able to access the following services:
 
-- Open edX LMS: http://localhost:8072
-- Open edX CMS: http://localhost:8082
+- Open edX LMS: http://localhost:8052
+- Open edX CMS: http://localhost:8062
```

Comment:
not sure it's necessary as you wouldn't want to run this in parallel with another instance would you?

Reply:
I did at some point